README.md

```diff
@@ -1,6 +1,52 @@
+# Welcome to Ansible Examples!
 
-Ansible Examples
-----------------
+First and foremost, for those who don't know, [Ansible](https://www.ansible.com)
+is a radically simple IT automation platform that makes your applications and
+systems easier to deploy. It allows you and your team to avoid writing scripts
+or custom code to deploy and update your applications, systems, and various
+classifications of network attached devices. Ansible allows you to automate in a
+language that approaches plain English, with no agents to install on remote
+systems and use native protocols based on device type such as SSH for
+UNIX-style operating systems, WinRM for Windows systems, REST APIs (httpapi)
+for REST API appliances, and many many more. Ansible is capable of speaking to
+any native protocol to any device.
 
-This repository contains examples and best practices for building Ansible Playbooks.
+If you are not familiar with Ansible and have arrived here looking for a place
+to start, please check out the [Getting
+Started
+Guide](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)
```

Comment:
The markdown does render correctly, but there's a lot of line breaks going on here.
Is playbook conditionals really what you meant to link to?